### PR TITLE
Add info about why VaultPress Backup is superior to hosts

### PR DESCRIPTION
**Pull request by simonktnga8c**
We're adding some copy to the plugin listing to highlight the benefits that VaultPress Backup provides over host backups

## Jetpack product discussion

p1HpG7-lOo

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Check for typos
* Preview the readme in https://wpreadme.com/

### CODE REVIEW

File: projects/plugins/backup/readme.txt [hunk 1]
```diff
@@ -28,6 +28,15 @@ If you make money from your site, or spend hours perfecting content, you need Wo
 * No developer required
 * Best-in-class support
 
+### If your host’s servers go down, you’re covered
+
+Host backups offer some protection but may not be sufficient to fully safeguard your valuable data. They often have limitations, such as single daily backups only, on-site storage, and clunky interfaces.
+
+Unlock the VaultPress Backup advantage:
+* Unmatched Security: Safeguard every change in real-time in the ultra-secure Jetpack Cloud
```
Comment (vianasw) [inline projects/plugins/backup/readme.txt]: Missing period here

**Review by vianasw — CHANGES_REQUESTED** [review]
Looking good. Just one minor change request.

**Review by vianasw — DISMISSED** [review]
LGTM 🚢

File: projects/plugins/backup/readme.txt [hunk 2]
```diff
@@ -28,6 +28,15 @@ If you make money from your site, or spend hours perfecting content, you need Wo
 * No developer required
 * Best-in-class support
 
+### If your host’s servers go down, you’re covered
+
+Host backups offer some protection but may not be sufficient to fully safeguard your valuable data. They often have limitations, such as single daily backups only, on-site storage, and clunky interfaces.
+
+Unlock the VaultPress Backup advantage:
+* Unmatched Security: Safeguard every change in real-time in the ultra-secure Jetpack Cloud.
```
Comment (jeherve) [inline projects/plugins/backup/readme.txt:37]: We need an empty line here so the list can be properly rendered:

```suggestion

* Unmatched Security: Safeguard every change in real-time in the ultra-secure Jetpack Cloud.
```

**Review by jeherve — CHANGES_REQUESTED** [review]
Just a small change, I'll commit it now.

For future reference, a good way to preview how the readme will appear on WordPress.org, you can use https://wpreadme.com/

**Review by jeherve — APPROVED** [review]

**Pull request merged**